app.yaml

```diff
@@ -30,6 +30,7 @@ handlers:
     Vary: Accept-Encoding
 - url: /build
   static_dir: build
+  application_readable: true
```

Comment:
Can this PR do the final solution in here, rather than having an interim period of inlining static files?